tests/e2e/linalg_ext_ops/BUILD.bazel

```diff
@@ -39,6 +42,43 @@ iree_check_single_backend_test_suite(
     target_backend = "llvm-cpu",
 )
 
+I1_SRCS = enforce_glob(
+    # keep sorted
+    [
+        "attention.mlir",
+        "attention_i1_mask.mlir",
+        "scan.mlir",
+        "scatter.mlir",
+        "sort.mlir",
+        "top-k.mlir",
+        "winograd_input.mlir",
+        "winograd_output.mlir",
+    ],
+    include = ["*.mlir"],
```

Comment:
can't we just include specifically the i1 related MLIR files since there are so less of them?

Reply:
@raikonenfnu at this moment it cannot because we need a new option to enable i1 packed storage for such test.

Reply:
How about something like
```cmake
iree_check_single_backend_test_suite(
    name = "check_llvm-cpu_local-task",
    srcs = [
        "attention_i1_mask.mlir",
    ],,
    compiler_flags = [
        "--iree-llvmcpu-target-cpu=generic",
        "--iree-experimental-packed-i1-storage",
    ],
    driver = "local-task",
    tags = [
        "nowasm",
    ],
    target_backend = "llvm-cpu",
)


```

Reply:
+1, would prefer to not add parallel lists of tests to run with experimental flags.

This also deviates from the existing structure where each list of test sources is for a specific backend+driver configuration. Should we have `VMVX_I1_SRCS`, `LLVM_GPU_I1_SRCS`, etc. as more backends work with the `--iree-experimental-packed-i1-storage` flag...?

What's the plan for that flag? Is it going to be removed when the behavior is stable and enabled by default?

Reply:
@ScottTodd @raikonenfnu Yeah the flag will eventually get removed and let the frontend to emit some attribute instead.